pages/common/gh-alias.md

```diff
@@ -0,0 +1,24 @@
+# gh alias
+
+> Manage GitHub command shortcuts from the command line.
+> More information: <https://cli.github.com/manual/gh_alias>.
+
+- Display the subcommand help:
+
+`gh alias`
+
+- List all of the aliases gh is configured to use:
+
+`gh alias list`
+
+- Set a command shortcut:
```

Comment:
I don't think I like the use of "shortcut" in favor of using the term "alias". Aliases are well known in our context so I would prefer having them as "alias". 
But since this is just personal preference and not an issue on the page, I'm approving the PR.
If anyone agrees with the alias thing please hop in.

Reply:
I think I agree with that, once you point it out.